source/opt/passes.h

```diff
@@ -58,6 +58,14 @@ class StripDebugInfoPass : public Pass {
   bool Process(ir::Module* module) override;
 };
 
+class FreezeSpecConstantValuePass : public Pass {
+ public:
+  const char* name() const override {
+    return "Freeze spec constants to their default value";
```

Comment:
We should have a rule that the name() is the same as the option used on the spirv-opt command line.